### PR TITLE
Feature: chatcommand assign [WIP]

### PR DESCRIPTION
**Pull request by goern**
add a chat command so that we can assign issues to users

### CODE REVIEW

File: aicoe/sesheta/chatbot.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    """Get the Intent of the provided text, and assign it a score."""
+    if text.startswith("assign"):
+        return ("assign", 1.0, {})
+
+    return (None, 0.0, {})
+
+
+async def process_user_text(thread_id: str, text: str) -> str:
+    """Process the Text, get the intent, and schedule actions accordingly."""
+    _LOGGER.info(f"message on thread {thread_id}: {text}")
```
Comment (fridex) [inline aicoe/sesheta/chatbot.py]: It's good practice (more optimal) to have arguments to logger. f-strings need to be expanded even when log-level turns off logging.

File: aicoe/sesheta/chatbot.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+from thoth.common import init_logging
+
+
+from aicoe.sesheta.utils import notify_channel, hangouts_userid, realname
+
+
+__version__ = "0.1.0-dev"
+
+
+init_logging()
```
Comment (fridex) [inline aicoe/sesheta/chatbot.py]: `THOTH_LOG_` will not work. A prefix argument needs to be passed.

File: aicoe/sesheta/chatbot.py (hunk 1 above)
Comment (goern) [reply aicoe/sesheta/chatbot.py]: after reading https://realpython.com/python-f-strings/#speed I thought it's not that much of an impact :P

File: aicoe/sesheta/chatbot.py (hunk 2 above)
Comment (goern) [reply aicoe/sesheta/chatbot.py]: added by 819e80c7eef5015c4a81b4aa2aa67d0e53be30b5

File: aicoe/sesheta/chatbot.py (hunk 1 above)
Comment (fridex) [reply aicoe/sesheta/chatbot.py]: I meant expansion directly in logger formatter which is lazy based on loglevel set:

```python
    _LOGGER.info("message on thread %s: %s", thread_id, text)
```

For example, pylint might treat this as a not good practice.

**Pull request merged**